### PR TITLE
add ffmpeg-normalize

### PR DESCRIPTION
**Pull request by slhck**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/ffmpeg-normalize/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/ffmpeg-normalize/meta.yaml:1]: Please delete comments that are not specific to this recipe.

File: recipes/ffmpeg-normalize/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  build:
+    - python
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tqdm >=4.38.0
+    - colorama >=0.4.3
+    - ffmpeg-progress-yield >=0.1.2
```
Comment (synapticarbors) [inline recipes/ffmpeg-normalize/meta.yaml:34]: It looks like `ffmpeg-progress-yield` isn't available via the conda-forge channel. You'll need to add it to get this to build without errors.

File: recipes/ffmpeg-normalize/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+build:
+  skip: true  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - ffmpeg-normalize = ffmpeg_normalize.__main__:main
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/ffmpeg-normalize/meta.yaml:25]: I don't think you need this `build` section

File: recipes/ffmpeg-normalize/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slhck/ffmpeg-normalize/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3c2fa6c37d91430d2cf2b52d45f450c8795a479cbad43961deece66f0542c136
+
+build:
+  skip: true  # [py<36]
```
Comment (synapticarbors) [inline recipes/ffmpeg-normalize/meta.yaml:18]: I believe you can drop the `skip` line and make this `noarch: python`. Just restrict `python` down in the requirements section under both `host` and `run` to `python >=3.6`

File: recipes/ffmpeg-normalize/meta.yaml (hunk 2 above)
Comment (slhck) [reply recipes/ffmpeg-normalize/meta.yaml:27]: I will create a separate PR for this. Thanks for letting me know.

**Pull request merged**